.github/workflows/nodejs.yml

```
@@ -36,5 +41,17 @@ jobs:
       run: npm i
 
     - name: Test
-      run: npm test
+      id: test
+      run: |
+        npm run test-coverage-ci || echo "Silently ignoring coverage threshold limit..."
+
+    - name: Comment test coverage report on PR
+      uses: romeovs/lcov-reporter-action@v0.2.16
+      with:
+        lcov-file: ./coverage/lcov.info
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

The original action used is based on a deprecated version of `@actions/core` that runs on Node 12.x & generates a deprecation warning[^1]. I haven't seen much activity on the repo and the issue was reported quite awhile ago[^2]. It may be better to use a different coverage reporting action before we merge this. 

Below is a suggested change using [zgosalvez/github-actions-report-lcov](https://github.com/zgosalvez/github-actions-report-lcov) instead. 
```suggestion
    - name: Report code coverage
      uses: zgosalvez/github-actions-report-lcov@v3
      with:
        coverage-files: ./coverage/lcov.info
        github-token: ${{ secrets.GITHUB_TOKEN }}
        update-comment: true
```

[^1]: https://github.blog/changelog/2023-06-13-github-actions-all-actions-will-run-on-node16-instead-of-node12-by-default/
[^2]: https://github.com/romeovs/lcov-reporter-action/issues/55